science/apbs/Portfile

```diff
@@ -2,11 +2,8 @@
 
 PortSystem              1.0
 PortGroup               compiler_blacklist_versions 1.0
-PortGroup               cmake 1.0
-PortGroup               mpi 1.0
-PortGroup               github 1.0
```

Comment:
Projects hosted at GitHub should use the github portgroup.

Reply:
Added github portgroup as requested.

Reply:
I see that you added the portgroup but I don't see it being used. Using the portgroup means calling `github.setup` with the right arguments.

Reply:
Updated the Portfile to use github.setup for the 3.4.1 tagged release of apbs.